fc-community-contracts/src/FactChainCommunity.sol

```diff
@@ -89,10 +89,21 @@ contract FactChainCommunity is Ownable, IFactChainCommunity {
     /// It costs an extra 20k gas per call to createNote/rateNote
     mapping(address => UserStats) public userStats;
 
+    /// @notice Mapping of note identifier (postUrl + creatorAddress) to NFT
+    mapping(string => mapping(address => uint256)) public nftIDs;
+
     /// @notice Instantiate a new contract and set its owner
     /// @param _owner Owner of the contract
     constructor(address _owner) Ownable(_owner) {}
 
+    ////////////////////////////////////////////////////////////////////////
+    /// Getter functions
+    ////////////////////////////////////////////////////////////////////////
+
+    function getNoteRaters(string memory _postUrl, address _creator) public view virtual returns (address[] memory) {
```

Comment:
let's try to write a simple contract that shows why this is needed, maybe there is solidity bug?

Reply:
maybe later alligator, or in a while crocodile 🐊